daliuge-common/dlg/clients.py

```diff
@@ -184,4 +184,11 @@ def __init__(self, host='localhost', port=constants.MASTER_DEFAULT_REST_PORT, ti
         super(MasterManagerClient, self).__init__(host=host, port=port, timeout=timeout)
 
     def create_island(self, island_host, nodes):
-        self._post_json('/managers/%s/dataisland' % (urllib.quote(island_host)), {'nodes': nodes})
\ No newline at end of file
+        self._post_json('/managers/%s/dataisland' % (urllib.quote(island_host)), {'nodes': nodes})
+
+class SimpleManagerClient():
+    """
+    A dummy manager client that only responds with a simple node list
+    """
+    def nodes(self):
+        return ['localhost', 'localhost']
```

Comment:
Not needed in reality (see comment in `pgtcwl_get` in `lg_web.py`)

Reply:
Agreed, I can do this

Reply:
I've removed this.